command/command_test.go

```diff
@@ -95,6 +95,9 @@ func TestPrepareWithCreate(t *testing.T) {
 	conf, mockHandlers, ts, deferFunc := newMockAPIServer(t)
 	defer deferFunc()
 
+	// To prevent unexpected behavior in the TravisCI environment
```

Comment:
If we don't have interests about cloud platforms (I suspect so, but not sure), can we set this in `newMockAPIServer`. How do you think?

Reply:
Well, I want to fix only the test cases that fall due to SSRF-like behavior, so I think it is better that inserted them in ad hoc.